webpack.config.js

```diff
@@ -28,25 +29,32 @@ const bundleEnv = config( 'env' );
 const isWindows = os.type() === 'Windows_NT';
 
 /**
- * This function scans the /client/extensions directory in order to generate a map that looks like this:
+ * This function scans the specified directory in order to generate an alias to path map that looks like this:
  * {
  *   sensei: 'absolute/path/to/wp-calypso/client/extensions/sensei',
  *   woocommerce: 'absolute/path/to/wp-calypso/client/extensions/woocommerce',
  *   ....
  * }
  *
- * Providing webpack with these aliases instead of telling it to scan client/extensions for every
+ * Providing webpack with these aliases instead of telling it to scan the directory for every
  * module resolution speeds up builds significantly.
+ *
+ * @param { String } directory - A directory to scan for modules.
+ * @param { String } prefix - A unique prefix for the alias to prevent alias collisions.
+ * @return { Object } aliasesMap - The alias to path map.
  */
-function getAliasesForExtensions() {
```

Comment:
I originally made the change so I could reuse it for the README assets, but I removed that logic, and this change remained.
At this point, there's no reason for me to have changed it, but at the same time, there's no reason to change it back so I'll leave it as is.